gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -92,7 +92,10 @@ def __init__(self,
             '--temp_location {}'.format(context.temp_location),
             '--job_name {}-{}'.format(test_name, dataset_id.replace('_', '-'))]
     for k, v in kwargs.iteritems():
-      args.append('--{} {}'.format(k, v))
+      if k == 'annotation_output_dir':
+        args.append('--{} {}/{}'.format(k, v, dataset_id))
```

Comment:
instead of special-casing this flag, consider adding "/{DATASET_ID}" to the string and always doing `.format(DATASET_ID=dataset_id)`.

Reply:
Done; good suggestion.
Actually your comment reminded me that dataset_id is not enough and this should actually be table_id because if we have multiple tests running VEP then they clash in output dir name. Fixed now.